index.bs

```diff
@@ -1702,6 +1702,34 @@ which might be expressed as a bitmask in another language,
 use a dictionary object instead.
 This object can be passed around as easily as a single bitmask value.
 
+<h3 id="async-by-default">If you need both asynchronous and synchronous methods, synchronous is the exception</h3>
```

Comment:
I think this is somewhat different than what is discussed in the issue, which is more nuanced. Are there no cases where sync is the common case and async the exception? I’m fine for us to encourage nudging things towards async when it could go either way, but I’m a little wary of a blanket “sync is always the exception”

Reply:
```suggestion
<h3 id="async-by-default">If you need both asynchronous and synchronous methods, prefer asynchronous</h3>
```